src/content/docs/en/reference/api-reference.mdx

```diff
@@ -788,6 +803,20 @@ export function GET({ locals }: APIContext) {
 
 See also: [`Astro.locals`](#astrolocals)
 
+### `context.routePattern`
+
+<Since v="5.0.0" />
```

Comment:
```suggestion
<p>

**Type:** `string` <br />
  <Since v="5.0.0" />
  </p>
```
Help me out again here!

Reply:
Looks good but (nit) we can remove the additional indents:
```suggestion
<p>

**Type:** `string` <br />
<Since v="5.0.0" />
</p>
```